recipes/easycrystallography/recipe.yaml

```diff
@@ -0,0 +1,46 @@
+context:
+  name: easycrystallography
+  version: 0.7.2
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/easycrystallography-${{ version }}.tar.gz
+  sha256: f30892982ab9180b996cee786b3ca2d8d10398e2e0d0125baf33a3a1454ca984
+
+build:
```

Comment:
Unless I'm mistaken, this library should be `noarch: python`:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

Reply:
Yes indeed! But this depends on `easyscience` which in turn depends on `scipp` which is currently only built for python 3.11 and python 3.12 on conda-forge. What's the best way to address this?

Maybe I can do a pyproject.toml patch to add python bounds?

Reply:
In `EasyCrystalloggraphy`'s `pyproject.toml`, it specifies: `requires-python = '>=3.9,<3.13'`. To get this to build you would do:

Here you would just do:

```
context:
   python_min: "3.11"
  ...

requirements:
   host:
      - python ${{ python_min }}.*
      ...
   run:
      - python >=${{ python_min }},<3.13
      ...
```

and then in `test`:

```
tests:
  - python:
      imports:
        - easycrystallography
      pip_check: true
      python_version: ${{ python_min }}.*
```